app/core/models/member.py

```diff
@@ -0,0 +1,44 @@
+import uuid
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class Member(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False,
+                          help_text=_("ID único seguindo o padrão UUID4.")
+                          )
+    user = models.ForeignKey('User', on_delete=models.CASCADE, verbose_name=_('usuário'),
+                             help_text=_('Usuário membro de um caderno')
+                             )
+    notebook = models.ForeignKey('Notebook', on_delete=models.CASCADE, verbose_name=_('caderno'),
+                                 help_text=_('Caderno no qual o usuário é membro'),
+                                 )
+
+    class Roles(models.TextChoices):
+        MEMBER = 'member', _('Member')
+        MODERATOR = 'mod', _('Moderator')
+        ADMIN = 'admin', _('Administrator')
```

Comment:
A linguagem padrão de internacionalização do projeto é português.

```suggestion
        MEMBER = 'member', _('membro')
        MODERATOR = 'mod', _('moderador')
        ADMIN = 'admin', _('administrador')
```